keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -714,12 +860,19 @@ message ProvisionerRevokeBucketAccessResponse {
 # Graduation Criteria
 
 ## Alpha
-
 - API is reviewed and accepted
 - Implement all COSI components to support Greenfield, Green/Brown Field, Brownfield and Static Driverless provisioning
-- Evaluate gaps, update KEP and conduct reviews for all design changes 
+- Evaluate gaps, update KEP and conduct reviews for all design changes
 - Develop unit test cases to demonstrate that the above mentioned use cases work correctly
 
+## Alpha -\> Beta
+- Basic unit and e2e tests as outlined in the test plan.
+- Metrics in kubernetes/kubernetes for bucket create and delete, and granting and revoking bucket access.
```

Comment:
I'm not sure it makes sense to add metrics to kubernetes for an out of tree component.

Reply:
agreed, this is just a vestigial remnant of our previous designs. It has been cleaned up now